src/TortoiseProc/GitLogListBase.cpp

```diff
@@ -95,8 +95,16 @@ CGitLogListBase::CGitLogListBase():CHintListCtrl()
 	m_bFilterWithRegex = !!CRegDWORD(_T("Software\\TortoiseGit\\UseRegexFilter"), FALSE);
 	m_bFilterCaseSensitively = !!CRegDWORD(_T("Software\\TortoiseGit\\FilterCaseSensitively"), FALSE);
 
-	m_From = -1;
-	m_To = -1;
+	m_Filter.m_NumberOfLogsScale = (DWORD)CRegDWORD(_T("Software\\TortoiseGit\\LogDialog\\NumberOfLogsScale"), -1);
+	m_Filter.m_NumberOfLogs = (DWORD)CRegDWORD(_T("Software\\TortoiseGit\\LogDialog\\NumberOfLogs"), 0);

```

Comment:
​​
>About your i1003-test branch:
I like the settings dialog

Thank you. :)

> There is no way for making the log dialog "remembering" the last selected date on close

Fixed. :)

>Why is the From date surrounded with a border when no limitation is set?

Obviously, my fault, no "if(){}" in OnPaint().
Fixed with no border there.

>Do we need all those options in the "Last x .." entries in the context menu?

Dropped them.

>Maybe add a checkbox entry in the context menu for "remember selected date on close"

Added.

>It does not compile ;) (just 2 minor c&p errors)

Oops. Also fixed.

----
All of all, thank you for pointing these out. appreciate. ^_^